web/src/components/AttributeList/AttributeList.tsx

```diff
@@ -1,17 +1,30 @@
-import {TSpanFlatAttribute} from '../../types/Span.types';
-import AttributeRow from '../AttributeRow';
+import AttributeRow from 'components/AttributeRow';
+import {IResultAssertions} from 'components/SpanDetail/SpanDetail';
+import {TSpanFlatAttribute} from 'types/Span.types';
 import * as S from './AttributeList.styled';
 
-interface IAttributeListProps {
+interface IProps {
+  assertions?: IResultAssertions;
   attributeList: TSpanFlatAttribute[];
   onCreateAssertion(attribute: TSpanFlatAttribute): void;
 }
 
-const AttributeList: React.FC<IAttributeListProps> = ({attributeList, onCreateAssertion}) => {
+const AttributeList: React.FC<IProps> = ({assertions, attributeList, onCreateAssertion}) => {
+  const onCopy = (value: string) => {
+    navigator.clipboard.writeText(value);
+  };
+
   return (
     <S.AttributeList>
       {attributeList.map(attribute => (
-        <AttributeRow attribute={attribute} key={attribute.key} onCreateAssertion={onCreateAssertion} />
+        <AttributeRow
+          assertionsFailed={assertions?.[attribute.key]?.failed}
+          assertionsPassed={assertions?.[attribute.key]?.passed}
```

Comment:
I would just pass the assertion as a prop.

Reply:
Thanks @cescoferraro 
I don't want to pass the whole object to the row if it's not needed. Here I'm looking for the `attribute.key` in the assertions object. With this we avoid having the assertions in every `AttributeRow`.